contracts/stabilizer/PriceBoundPSM.sol

```diff
@@ -0,0 +1,114 @@
+pragma solidity ^0.8.4;
+
+import "./PegStabilityModule.sol";
+import "./IPriceBoundPSM.sol";
+import "../Constants.sol";
+
+/// @notice contract to create a price bound DAI PSM
+/// This contract will allow swaps when the price of DAI is between 98 cents and 1.02 by default
+/// These defaults are changeable by the admin and governance by calling floor and ceiling setters
+/// setOracleFloor and setOracleCeiling
+contract PriceBoundPSM is PegStabilityModule, IPriceBoundPSM {
+    using Decimal for Decimal.D256;
+    using SafeERC20 for IERC20;
+    using SafeCast for *;
+
+    /// @notice get the basis points delta
+    uint256 constant public override bpDelta = 200;
```

Comment:
Do we want this to be symmetric? Unclear and prefer separate floor and ceiling

Reply:
I can remove this as the purpose of it is to stop us from buying when DAI loses its peg.